akka-actor-typed/src/main/scala/akka/actor/typed/internal/PoisonPill.scala

```diff
@@ -12,6 +12,9 @@ import akka.annotation.InternalApi
 
 /**
  * INTERNAL API
+ *
+ * Note that this is not a universal poison pill like the untyped actor one but requires special handling
+ * on the receiving side where it is used (for example with the interceptor below.
  */
 @InternalApi private[akka] sealed abstract class PoisonPill extends Signal
```

Comment:
I'm sure not, for several reasons, but bringing up anyway?
```scala
@InternalApi private[typed] sealed abstract class PoisonPill extends Signal
```

Reply:
Yeah, I'd probably have done narrower access control but Akka tradition [tm] is to keep them project private rather.